R/dfm.R

```diff
@@ -28,6 +28,7 @@
 #'   separately, or using [tokens_lookup()] on the tokenized text
 #'   before calling `dfm`.
 #' @inheritParams valuetype
+#' @param case_insensitive ignore the case of dictionary values if `TRUE`
```

Comment:
will this affect matches for patterns specified with `select` as well?

Reply:
This is a wrong description that I compiled from other function...

Reply:
We should probably review this across functions for consistency. I just noticed that `dfm_select()` also refers to dictionaries.

If we add this to the man page for valuetype we could put it all in one place, and @inheritParams when needed as with valuetype. I would describe then as:
```r
@param case_insensitive logical; if `TRUE`, ignore case when matching a `pattern` [dictionary] values
```